pom.xml

```diff
@@ -854,18 +854,17 @@
         <version>1.7.8</version>
       </dependency>
 
-      <!-- Databased stuff -->
-      <dependency>
-        <groupId>com.h2database</groupId>
-        <artifactId>h2</artifactId>
-        <version>1.3.174</version>
-      </dependency>
       <!-- Other stuff -->
       <dependency>
         <groupId>dlib</groupId>  <!--FIXME Handled by local repository -->
         <artifactId>dlib</artifactId>
         <version>1.0</version> <!-- FIXME totally unknown -->
       </dependency>
+      <dependency>
+        <groupId>org.mapfish.print</groupId>
+        <artifactId>print-lib</artifactId>
+        <version>${print-lib.version}</version>
```

Comment:
This variable may not be needed, unless folks are in the habit of specifying it on the command line use `-Dprint-lib.version` ?

Reply:
isn't' it defined in main `pom.xml? 
https://github.com/geonetwork/core-geonetwork/blob/55704fae0bccf7a167fd4dc8c666d5c0c4d40858/pom.xml#L1452

Reply:
It is defined, but since we only need to use this in the dependency management section once ... why bother with the complexity?